linker/shared/src/main/scala/org/scalajs/linker/backend/emitter/SJSGen.scala

```diff
@@ -116,13 +116,18 @@ private[emitter] final class SJSGen(
 
   def getArrayUnderlyingTypedArrayClassRef(elemTypeRef: NonArrayTypeRef)(
       implicit pos: Position): Option[WithGlobals[VarRef]] = {
-
-    def some(name: String): Some[WithGlobals[VarRef]] =
-      Some(WithGlobals(VarRef(Ident(name)), Set(name)))
-
     elemTypeRef match {
       case _ if !esFeatures.useECMAScript2015 => None
+      case primRef: PrimRef                   => typedArrayRef(primRef)
+      case _                                  => None
+    }
+  }
+
+  def typedArrayRef(primRef: PrimRef)(
```

Comment:
I think this change (the extraction of `typedArrayRef` is in the wrong commit. IIUC it should be in the last commit instead.

Reply:
Yeah, I noticed that as well in self-review :)